src/lrs.rs

```diff
@@ -181,24 +196,30 @@ impl<CurveImpl: Curve> Lrs<CurveImpl> {
             });
         }
 
+        println!("iterations on lrm scales");
```

Comment:
is it intended to keep this print ? if yes, ignore this comment

Reply:
Not, not at all, thank you :)